src/facade/dragonfly_connection.cc

```diff
@@ -249,6 +252,9 @@ constexpr size_t kMinReadSize = 256;
 
 thread_local uint32_t free_req_release_weight = 0;
 
+// A cached value from FLAGS_pipeline_queue_limit.
+__thread uint32_t pipeline_queue_limit_cached = 256;
+
```

Comment:
It could've been added to QueueBackpressure struct (that already caches memory limits and uses IsPipelineBufferOverLimit) and moved the logic there instead of complicating connection code. ` Connection::SetMaxQueueLenThreadLocal` also wouldn't be needed

Reply:
why SetMaxQueueLenThreadLocal would not be needed?
how I would change the flag?

Reply:
It would be needed on the backpressure class, but wouldn't pollute connection

Reply:
backpressure class is private to connection so I still need to expose `Connection::SetMaxQueueLenThreadLocal`